### PR TITLE
jwt 통한 회원 인증 및 인가 (RE)

### PR DESCRIPTION
**Pull request by EunjungOh123**
controller까지 추가 구현

### CODE REVIEW

**Comment by naekang** [comment]
|File|Coverage [40.62%]|:x:|
|:-|:-:|:-:|
|[MemberErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/ecb32bf1dd7e1b97a02ae87c7a413a167e4976e6/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fmember%2Fexception%2Ferrorcode%2FMemberErrorCode.java)|100%|:green_apple:|
|[CustomAccessDeniedHandler.java](https://github.com/Gaethering/Gaethering-Server/blob/ecb32bf1dd7e1b97a02ae87c7a413a167e4976e6/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Futil%2FCustomAccessDeniedHandler.java)|43.75%|:x:|
|[CustomAuthenticationEntryPoint.java](https://github.com/Gaethering/Gaethering-Server/blob/ecb32bf1dd7e1b97a02ae87c7a413a167e4976e6/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Futil%2FCustomAuthenticationEntryPoint.java)|43.75%|:x:|
|[MemberServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/ecb32bf1dd7e1b97a02ae87c7a413a167e4976e6/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fmember%2Fservice%2FMemberServiceImpl.java)|42.92%|:x:|
|[MemberController.java](https://github.com/Gaethering/Gaethering-Server/blob/ecb32bf1dd7e1b97a02ae87c7a413a167e4976e6/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fmember%2Fcontroller%2FMemberController.java)|39.78%|:x:|
|[RedisUtil.java](https://github.com/Gaethering/Gaethering-Server/blob/ecb32bf1dd7e1b97a02ae87c7a413a167e4976e6/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Futil%2FRedisUtil.java)|16.36%|:x:|
|[JwtAuthenticationFilter.java](https://github.com/Gaethering/Gaethering-Server/blob/ecb32bf1dd7e1b97a02ae87c7a413a167e4976e6/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Ffilter%2FJwtAuthenticationFilter.java)|14.63%|:x:|
|[CustomUserDetailsService.java](https://github.com/Gaethering/Gaethering-Server/blob/ecb32bf1dd7e1b97a02ae87c7a413a167e4976e6/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fmember%2Fservice%2FCustomUserDetailsService.java)|12%|:x:|
|[JwtProvider.java](https://github.com/Gaethering/Gaethering-Server/blob/ecb32bf1dd7e1b97a02ae87c7a413a167e4976e6/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Futil%2FJwtProvider.java)|8.5%|:x:|

|Total Project Coverage|45.18%|:x:|
|:-|:-:|:-:|

File: src/main/java/com/gaethering/gaetheringserver/member/dto/LoginRequest.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.gaethering.gaetheringserver.member.dto;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
```
Comment (younghoondoodoom) [inline src/main/java/com/gaethering/gaetheringserver/member/dto/LoginRequest.java:9]: 혹시 NoArgsConstructor는 어디서 사용되나요??

**Review by younghoondoodoom — COMMENTED** [review]
고생하셨습니다! 최대한 빠르게 머지해주시면 감사하겠습니다😀

**Comment by naekang** [comment]
|File|Coverage [40.62%]|:x:|
|:-|:-:|:-:|
|[MemberErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/605bc955bd8a454cfb51e082b014c5c81acd3dbd/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fmember%2Fexception%2Ferrorcode%2FMemberErrorCode.java)|100%|:green_apple:|
|[CustomAccessDeniedHandler.java](https://github.com/Gaethering/Gaethering-Server/blob/605bc955bd8a454cfb51e082b014c5c81acd3dbd/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Futil%2FCustomAccessDeniedHandler.java)|43.75%|:x:|
|[CustomAuthenticationEntryPoint.java](https://github.com/Gaethering/Gaethering-Server/blob/605bc955bd8a454cfb51e082b014c5c81acd3dbd/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Futil%2FCustomAuthenticationEntryPoint.java)|43.75%|:x:|
|[MemberServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/605bc955bd8a454cfb51e082b014c5c81acd3dbd/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fmember%2Fservice%2FMemberServiceImpl.java)|42.92%|:x:|
|[MemberController.java](https://github.com/Gaethering/Gaethering-Server/blob/605bc955bd8a454cfb51e082b014c5c81acd3dbd/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fmember%2Fcontroller%2FMemberController.java)|39.78%|:x:|
|[RedisUtil.java](https://github.com/Gaethering/Gaethering-Server/blob/605bc955bd8a454cfb51e082b014c5c81acd3dbd/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Futil%2FRedisUtil.java)|16.36%|:x:|
|[JwtAuthenticationFilter.java](https://github.com/Gaethering/Gaethering-Server/blob/605bc955bd8a454cfb51e082b014c5c81acd3dbd/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Ffilter%2FJwtAuthenticationFilter.java)|14.63%|:x:|
|[CustomUserDetailsService.java](https://github.com/Gaethering/Gaethering-Server/blob/605bc955bd8a454cfb51e082b014c5c81acd3dbd/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fmember%2Fservice%2FCustomUserDetailsService.java)|12%|:x:|
|[JwtProvider.java](https://github.com/Gaethering/Gaethering-Server/blob/605bc955bd8a454cfb51e082b014c5c81acd3dbd/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Futil%2FJwtProvider.java)|8.5%|:x:|

|Total Project Coverage|45.18%|:x:|
|:-|:-:|:-:|

**Pull request merged**